src/main/java/net/sf/jabref/gui/errorconsole/ErrorConsoleView.java

```diff
@@ -80,14 +90,91 @@ private void copyLogButton() {
 
     @FXML
     private void createIssueButton() {
-        errorViewModel.createIssue();
+        errorViewModel.reportIssue();
     }
 
-    // handler for close of error console
     @FXML
     private void closeErrorDialog() {
         Stage stage = (Stage) closeButton.getScene().getWindow();
         stage.close();
     }
 
+    /*
+     * create a grid pane with two columns to insert one image on left side and one text on right side.
+     * This will be late set in the header of the dialog pane
+     * @return the generate grid pane
+     */
+    private GridPane createDialogPaneHeader() {
+        GridPane headerGrid = new GridPane();
+        ColumnConstraints graphicColumn = new ColumnConstraints();
+        graphicColumn.setFillWidth(false);
+        graphicColumn.setHgrow(Priority.NEVER);
+        ColumnConstraints textColumn = new ColumnConstraints();
+        textColumn.setFillWidth(true);
+        textColumn.setHgrow(Priority.ALWAYS);
+        headerGrid.getColumnConstraints().setAll(graphicColumn, textColumn);
+        headerGrid.setPadding(new Insets(10));
+
+        Image image = new Image("https://cdn2.iconfinder.com/data/icons/windows-8-metro-style/512/console.png");
```

Comment:
Don't use graphics from external resources such as urls. Have a look at #1390 to see how to use the material design icons in JavaFX.